healthcare/2.22/modules/ROOT/pages/index.adoc

```diff
@@ -16,7 +16,7 @@ The following describes the use cases supported by the MuleSoft Accelerator for
 * https://www.anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%205%20-%20Benefits%20and%20eligibility%20verification/[Benefits and eligibility verification^] — Enables call center agents and care staff to perform real-time benefits and eligibility verification. Providers can electronically verify a patient's insurance coverage in real-time for medical treatment directly within Salesforce Health Cloud.
 * https://www.anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%206%20-%20Prior%20Authorization/[Prior Authorization^] — Includes CRD, DTR, and PAS API specifications and implementation templates to automate the prior authorization process for more immediate authorizations, leading to better outcomes and increased satisfaction for patients, providers, and payers.
 * https://www.anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%207%20-%20Population%20health%20management/[Population health management^] -  Convert and surface valuable population health data into Amazon HealthLake to improve clinical outcomes.
-* https://www.anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%208%20-%20Qualified%20Health%20Information%20Networks%20integration/[Qualified Health Information Networks Integration] - Streamline patient onboarding and remove manual data entry by integrating with external healthcare organizations to gather and ingest patient data.
+* https://www.anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-healthcare/minor/{hls-version}/pages/Use%20case%208%20-%20Qualified%20health%20information%20networks%20integration/[Qualified Health Information Networks Integration] - Streamline patient onboarding and remove manual data entry by integrating with external healthcare organizations to gather and ingest patient data.
```

Comment:
This link still doesn't work for me. I think it's:
https://www.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-healthcare/minor/2.22/pages/57n-39f/Use%20case%208%20-%20Qualified%20health%20information%20networks%20integration/

Reply:
I think it's because you don't have the version in the URL--it's a placeholder {hls-version}